tests/test_core.py

```diff
@@ -71,9 +69,9 @@ def test_get_blocks_status_containing_tx(monkeypatch):
         {'id': 1}, {'id': 2}
     ]
     monkeypatch.setattr(
-        Bigchain, 'search_block_election_on_index', lambda x, y, z: blocks)
+        Bigchain, 'search_block_election_on_index', lambda x, y: blocks)
     monkeypatch.setattr(
-        Bigchain, 'block_election_status', lambda x, y: Bigchain.BLOCK_VALID)
+        Bigchain, 'block_election_status', lambda x, y, z: Bigchain.BLOCK_VALID)
```

Comment:
I wonder if it's worth just changing these lambdas to be `lambda *args, **kwargs` instead since they throw away the input?